aggregate/README.md

```diff
@@ -0,0 +1,65 @@
+# Aggregate Script
+
+## **Description**
+- The **Aggregate** script integrates protein-protein interaction data from databases Mentha and STRING into a single aggregated CSV file of ranked interactors. 
+- The output includes confidence scores of interaction along with available structures from the Protein Data Bank (PDB) or the Beltrao’s dataset.
+- The interactors are ranked based on the Mentha confidence score whereas STRING confidence score is the secondary ranking criterion.
+
+## **Requirements**
+### **Software:**
+- Python 3.x 
+- Python packages:
+  - `pandas`
+
+### **Files:**
+- A Mentha2PDB (`/data/user/shared_projects/mavisp/mavisp_templates_2024/GENE_NAME/interactome/mentha2pdb/mentha2pdb.py`) output CSV.
+- A STRING2PDB (`/data/user/shared_projects/mavisp/mavisp_templates_2024/GENE_NAME/interactome/string2pdb/string2pdb`) output CSV.

```

Comment:
they need to refer to the same target protein - better specify so in the documentation. Should be obvious, but not necessarily for a naive user :) 

also can you please remove these paths? They only work on our server, while the example should hold by itself (in general, it should work as is on any computer)